src/crypto/crypto_rsa.cc

```diff
@@ -210,7 +210,10 @@ WebCryptoCipherStatus RSA_Cipher(
   if (label_len > 0) {
     void* label = OPENSSL_memdup(params.label.get(), label_len);
     CHECK_NOT_NULL(label);
-    if (EVP_PKEY_CTX_set0_rsa_oaep_label(ctx.get(), label, label_len) <= 0) {
+    if (EVP_PKEY_CTX_set0_rsa_oaep_label(
+      ctx.get(),
+      reinterpret_cast<unsigned char*>(label),
```

Comment:
```suggestion
      static_cast<unsigned char*>(label),
```

Reply:
What exactly is the difference here?

Reply:
@bl-ue `static_cast` works here, so we should prefer using that here.
We use `reinterpret_cast` only when it is essential to do something like:
```cpp
* (<new-type>*) &<old-type-variable>
```
Refs: https://github.com/nodejs/node/blob/master/doc/guides/cpp-style-guide.md#type-casting

Reply:
The important bit is that `static_cast` lets you do fewer “obviously wrong” things, like casting between unrelated classes – for `void*`, the cast is going to end up doing the exact same thing in the end, that’s true. 

Maybe think of `reinterpret_cast` as always coming with a little warning sign when reading the code – it means something’s happening here which requires an extra look to make sure that it’s actually doing the right thing.